libs/dh/wholesale/feature-start/src/lib/dh-wholesale-start.component.ts

```diff
@@ -142,6 +148,46 @@ export class DhWholesaleStartComponent implements OnInit, OnDestroy {
     });
   }
 
+  private mapGridAreasToDropdownOptions(
+    gridAreas: GridAreaDto[]
+  ): WattDropdownOption[] {
+    return (
+      gridAreas?.map((gridArea) => {
+        return {
+          displayValue: `${gridArea?.name} (${gridArea?.code})`,
+          value: gridArea?.code,
+        };
+      }) || []
```

Comment:
nitpick: again, function definition says it is never undefined
```suggestion
      gridAreas.map((gridArea) => {
        return {
          displayValue: `${gridArea?.name} (${gridArea?.code})`,
          value: gridArea?.code,
        };
      })
```